srcpkgs/clex/template

```diff
@@ -0,0 +1,23 @@
+# Template file for 'clex'
+pkgname=clex
+version=4.6.patch9
+revision=1
+archs="x86_64 i868 x86_64-musl"
+build_style="gnu-configure"
+configure_args="--prefix=/usr --sysconfdir=/etc --infodir=/usr/share/info --mandir=/usr/share/man --localstatedir=/var"
+hostmakedepends="ncurses-devel ncurses"
+short_desc="File manager with a full-screen user interface"
+maintainer="ThatGeekyWeeb <thatgeekyweeb@gmail.com>"
+license="GPL-2.0-or-later"
+homepage="http://www.clex.sk/"
+distfiles="http://www.clex.sk/download/clex-${version}.tar.gz"
+checksum="4c866d9d7bb4e276f1c54fb897d71f1d6390ec1535ffb174798a8334d1f545e2"
+do_build() {
+	./configure ${configure_args}
```

Comment:
configure is already called by the build-style in the do_configure step, remove it here